src/functions.js

```diff
@@ -219,14 +219,67 @@ var _ = Mavo.Functions = {
 	 * Min of an array of numbers
 	 */
 	min: function(array) {
-		return Math.min(...$u.numbers(array, arguments));
+		if (arguments.length > 1) {
+			var ret = arguments[0];
+			for (let i = 0; i < arguments.length; i++) {
+				var b = arguments[i] || null;
+				ret = Mavo.Script.binaryOperation(ret, b, {
+					scalar: (ret, b) => Math.min(ret, b),
+					identity: Infinity
+				});
+			}
+			return ret;
+		}
+		else {
+			return Math.min(...$u.numbers(array, arguments));
+		}
 	},
 
 	/**
 	 * Max of an array of numbers
 	 */
 	max: function(array) {
-		return Math.max(...$u.numbers(array, arguments));
+		if (arguments.length > 1) {
```

Comment:
This seems to be like a separate change, it's not about allowing arguments to be arrays. Also, it seems that multiple arguments in `min()` and `max()` already work, so I'm not sure what this change does exactly?